schema/schema_validator.py

```diff
@@ -141,17 +144,20 @@ def validate_test_data_with_schema(self):
         for test_type in self.test_types:
             for icu_version in self.icu_versions:
                 file_path_pair = self.get_schema_data_info(icu_version, test_type)
+                logging.info(' FILE PATH PAIR: %s', file_path_pair);
                 if file_path_pair:
                     schema_test_info.append(file_path_pair)
                 else:
                     test_data_files_not_found.append([icu_version, test_type])
-                    logging.debug('No data test file  %s for %s, %s', file_path_pair, test_type, icu_version)
+                    logging.debug('%s: No data test file  %s for %s, %s',
+                                  self.schema_type,
+                                  file_path_pair, test_type, icu_version)
                     pass
 
         if test_data_files_not_found:
             logging.info('Note: %d potential test data sets were not found.', len(test_data_files_not_found))
 
-        results = self.parallel_check_test_data_schema(schema_test_info)
+        results = self.check_all_test_data_schema(schema_test_info)
 
         for result_data in results:
```

Comment:
can replace the instantiation, for loop, and if with an FP-style `map` + `filter`.